addReleaseTag

```diff
@@ -105,6 +105,50 @@ response=`curl -s -X GET \
 response=`echo $response | jq .isLatest`
 
 
+# all files in the bundle directory have the same hash
+# so we can just take the first one and extract the hash
```

Comment:
thats very strange. I dont think this should be the case. The hash is computed based on the contents of the bundle? ANd each bundle should have different content? Maybe i am wrong

Reply:
Yes, please use file content based hash.

Reply:
done